### ISSUE TITLE
onScrollStart triggered even if no actual scrolling is triggered when using snapAmount

### ISSUE THREAD

**Issue by thibautsapient** [malihu/malihu-custom-scrollbar-plugin#442]
Hi,

The subject says it all :-)

If you use a snapAmount, because you need to "drag" the handle past his own height, onScrollStart is triggered as soon as you start dragging the handle. It will then scroll to the same position, but the even is still fired.

The expected behaviour is to trigger onScrollStart only if the element effectively scrolls.

Cheers

**Comment by malihu** [comment]
Yes indeed this happens with the `snapAmount` option. I'll have to test this further and see what can be done. In the meantime, you can use something like the code below to do what you describe: 

```
callbacks:{
	onScrollStart:function(){ this["mcsFlag"]=this.mcs.top+1; },
	whileScrolling:function(){ 
		if(this.mcsFlag && this.mcsFlag!==this.mcs.top+1){
			this.mcsFlag=null;
			/* do something when content starts moving */
		}
	}
}
```

Thanks for the feedback :)

**Comment by thibautsapient** [comment]
Thanks @malihu, great minds think alike! I was indeed planning to use something similar with the scrolling event while secretly hoping for a fix to be available :-)

Cheers
